mmcv/ops/csrc/pytorch/cuda/stack_group_points_cuda.cu

```diff
@@ -0,0 +1,100 @@
+// Copyright (c) OpenMMLab. All rights reserved.
+// Modified from
+// https://github.com/sshaoshuai/Pointnet2.PyTorch/tree/master/pointnet2/src/group_points_gpu.cu
+#include <stdio.h>
+#include <stdlib.h>
+
+#include "stack_group_points_cuda_kernel.cuh"
+#include "pytorch_cuda_helper.hpp"
+#define DIVUP(m,n) ((m) / (n) + ((m) % (n) > 0))
+#define CHECK_CUDA(x) do { \
```

Comment:
Why do we copy the macro here? we have the very same macro in `common/pytorch_cuda_helper.hpp`